infra-as-code/bicep/modules/hubNetworking/hubNetworking.bicep

```diff
@@ -235,6 +235,30 @@ param parTags object = {}
 @sys.description('Set Parameter to true to Opt-out of deployment telemetry.')
 param parTelemetryOptOut bool = false
 
+@sys.description('Define inbound destination port or a range for HTTPS. Default:443')
+param parHttpsPort string = '443'
+
+@sys.description('Define inbound destination port or a range for gateway manager. Default:443')
+param parGatewayManagerPort string  ='443'
+
+@sys.description('Define inbound destination port or a range for load balancer. Default:443')
+param parLoadBalancerPort string = '443'
+
+@sys.description('Define inbound destination ports or ranges for Bastion Host Communication. Default:[8080, 5701]')
+param parBastionHostCommunicationPorts array = ['8080','5701']
+
+@sys.description('Define outbound destination ports or ranges for SshRDPOutbound. Default:[22,3389]')
+param parSshRDPPort array = ['22','3389']
+
+@sys.description('Define outbound azure cloud destination port range for Azure cloud. Default:443')
+param parAzureCloudPort string = '443'
+
+@sys.description('Define outbound destination port ranges for Bastion Communication. Default:[8080, 5701]')
+param parBastionCommunicationPorts array = ['8080','5701']
```

Comment:
Duplicate + These ports cannot be changed on the bastion service, so they dont need to be exposed via a parameter.

Propose we remove and hardcode